doc/UsersGuide/source/InputsOutputs.rst

```diff
@@ -738,41 +738,104 @@ GOCART
+   * - NI2G_instance_NI.rc
+     - Resource file for Nitrate parameters               
+   * - SS2G_instance_SS.rc
+     - Resource file for Sea Salt parameters
+   * - SU2G_instance_SU.rc
+     - Resource file for Sulfur parameters
+
+GOCART inputs defined in AERO_ExtData are listed and described in :numref:`Table %s <GOCART_InputFiles>`.
+
+.. _GOCART_InputFiles:
+
+.. list-table:: *GOCART inputs defined in AERO_ExtData.rc*
    :widths: 40 50
    :header-rows: 1
 
    * - Filename
      - Description
-   * -
-     -
+   * - ExtData/dust 
+     - FENGSHA input files
+   * - ExtData/QFED
+     - QFED biomass burning emissions     
+   * - ExtData/CEDS
+     - Anthropogenic emissions
+   * - ExtData/MERRA2
+     - DMS concentration
+   * - ExtData/PIESA/sfc
+     - Aviation emissions
+   * - ExtData/PIESA/L127
+     - H2O2, OH and NO3 mixing ratios     
+   * - ExtData/MEGAN_OFFLINE_BVOC
+     - VOCs MEGAN biogenic emissions
+   * - ExtData/monochromatic
+     - aerosol monochromatic optics files 
+   * - ExtData/optics
+     - aerosol radiation bands optic files for RRTMG
+   * - ExtData/volcanic
+     - SO2 volcanic pointwise sources
+     
+The static input files when using climatology (MERRA2) are listed and described in :numref:`Table %s <Climatology_InputFiles>`.
+
+.. _Climatology_InputFiles:
+
+.. list-table:: *Inputs when using climatology (MERRA2)*
+   :widths: 40 50
+   :header-rows: 1
 
+   * - Filename
+     - Description             
+   * - merra2.aerclim.2003-2014.m$(month).nc
+     - MERRA2 aerosol climatology mixing ratio 
+   * - Optics_BC.dat
+     - BC optical look-up table for MERAA2 
+   * - Optics_DU.dat
+     - DUST optical look-up table for MERAA2
+   * - Optics_OC.dat
+     - OC optical look-up table for MERAA2
+   * - Optics_SS.dat
+     - Sea Salt optical look-up table for MERAA2
+   * - Optics_SU.dat
+     - Sulfate optical look-up table for MERAA2
+                    
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Grid description and initial condition files
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. TODO:: GOCART information in progress
-
-The input files containing grid information and the initial conditions for global configurations are listed and described in :numref:`Table %s <GOCART_GridICFiles>`.
-
-.. _GOCART_GridICFiles:
-
-.. list-table:: *Input files containing grid information and initial conditions for global configurations*
-   :widths: 35 50 15
-   :header-rows: 1
+Running GOCART in UFS does not require aerosol initial conditions, as aerosol models can always start from scratch (cold start), however, this approach does require more than two weeks of model spin-up to obtain reasonable aerosol simulation results. Therefore, the most popular method is to take previous aerosol simulation results. The result is not necessarily from the same model, it could be from a climatology result, such as MERAA2, or a different model but with the same aerosol species and bin/size distribution.
```

Comment:
I would suggest you break the first line into two sentences:
```
Running GOCART in UFS does not require aerosol initial conditions, as aerosol models can always start from scratch (cold start). However,.....
```

Reply:
changed it